secretsmanager/src/aes.cpp

```diff
@@ -31,11 +30,11 @@ vector<uint8_t> AES_CBC::encrypt(const string& input) {
       return {};
     }
 
-    unique_ptr<unsigned char[]> ciphertext(new unsigned char[input.size() + AES_BLOCK_SIZE]);
-    unique_ptr<unsigned char[]> plaintext(new unsigned char[input.size() + 1]);
+    auto ciphertext = std::make_unique<unsigned char[]>(input.size() + AES_BLOCK_SIZE);
+    auto plaintext = std::make_unique<unsigned char[]>(input.size() + 1);
 
     for (size_t i{0UL}; i < input.size(); ++i) {
-      plaintext.get()[i] = (unsigned char)input[i];
+      plaintext.get()[i] = static_cast<unsigned char>(input[i]);
```

Comment:
1) Not sure why plaintext is +1 ? std::string has no null terminator.
2) I suggest to change 36 to 38 to:
`std::copy(input.begin(), input.end(), plaintext.get());`

Reply:
Addressed in https://github.com/vmware/concord-bft/pull/2851/commits/d4b7969f8e0646cedea6cc7a8f9b76c8913ffee4